pkg/auth/rhsso_authz_handler_test.go

```diff
@@ -93,6 +97,187 @@ var _ = Describe("Authz email domain", func() {
 	}
 })
 
+var _ = Describe("OwnedBy", func() {
+	var (
+		db      *gorm.DB
+		dbName  string
+		ctx     context.Context
+		handler Authorizer
+		records []common.Cluster
```

Comment:
I think passing the records to validate to the `AllRecords*` functions would be better.
Sharing this variable across tests could lead to contamination.

You can do this in a followup or when it becomes a problem.

Reply:
very well. I localized it.